app/lib/features/invite_members/pages/invite_space_members.dart

```diff
@@ -63,28 +63,31 @@ class _InviteSpaceMembersConsumerState
     );
   }
 
-  Widget _buildParentSpace() {
-    final parentSpace =
-        ref.watch(canonicalParentProvider(widget.roomId)).valueOrNull;
+  Widget _buildParentSpaces() {
+    final parentSpaces =
+        ref.watch(canonicalParentsProvider(widget.roomId)).valueOrNull;
 
-    if (parentSpace == null) return const SizedBox.shrink();
+    if (parentSpaces == null && parentSpaces!.isEmpty) {
+      return const SizedBox.shrink();
+    }
 
     return Column(
       crossAxisAlignment: CrossAxisAlignment.start,
-      children: [
+      children: <Widget>[
         Text(L10n.of(context).parentSpace),
```

Comment:
this should probably also be updated ...
```suggestion
        Text(parentSpaces.length > 1 ? L10n.of(context).parentSpaces : L10n.of(context).parentSpace),
```